api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -388,6 +398,29 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
     dbWorkspace.addWorkspaceUserRole(permissions);
 
     dbWorkspace = workspaceService.getDao().save(dbWorkspace);
+
+    JSONObject demoCohort = cloudStorageService.readDemoCohort();
+    // TODO: Make this a service method to avoid duplication with CohortsController
+    Cohort dbCohort = new Cohort();
+    dbCohort.setName(demoCohort.getString("name"));
+    dbCohort.setDescription(demoCohort.getString("description"));
+    dbCohort.setCriteria(demoCohort.get("criteria").toString());
+    dbCohort.setType("AoU_Discover");
```

Comment:
Where did this string come from and what does it mean?

Reply:
Currently, nothing. I believe its for future proofing. However, I realized it may actually be useful for us here, so I should have something for it soon. (It is set hard coded in the UI right now on all cohorts)